boards/m5stack/m5stack_cores3/Kconfig.defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Zhang Xingtao <zhxt@live.cn>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_M5STACK_CORES3_ESP32S3_PROCPU
```

Comment:
The CoreS3-SE is available, which is a reduced feature version of CoreS3.
The  CoreS3 includes all CoreS3-SE features, so I want to use CoreS3-SE as a variant of CoreS3.
I think this can be done by including the dts file and adding additional functions to the CoreS3 side.

Please see 
https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/ee3bb8a72a524438e5d1e0ff77d35c0e723656ba
https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/1aa3e9713482589e509b89bb56f72e56241bdfbc
https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/56bb52f59b2622aafdc1c092b9e4ff9638e0f022

I could handle this myself, but I think it would be better to decide on the folder and file structure as early as possible.

Reply:
I reconsidered it would be better to handle by my PR so that I will withdraw it.

Reply:
> I reconsidered it would be better to handle by my PR so that I will withdraw it.

Okay.